neptune/internal/channels/channels.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import time
+
+
+class ChannelWithValue(object):
```

Comment:
This class seems to be useless since it's used only in `Experiment` implementation to pass value from one method to another. Moreover, its name is misleading. It does not contain a channel and a value. It's some tuple containing a name, a type and some other variables.
Just pass `channel_name`, `channel_type` and `value` of type `ChannelValue` separately in `Experiment` implementation.